asistente_ladm_col/logic/quality_rules/qr_party_duplicate_records.py

```diff
@@ -0,0 +1,97 @@
+"""
+/***************************************************************************
+                              Asistente LADM_COL
+                             --------------------
+        begin                : 2022-06-15
+        git sha              : :%H$
+        copyright            : (C) 2022 by Sergio Ramírez (BSF Swissphoto)
+        email                : seralra96@gmail.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames, LayerConfig
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4019,
+                                                           QRE_IGACR4019E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRDuplicatePartyRecords(AbstractLogicQualityRule):
+    """
+    Check that party don't have duplicate records
+    """
+    _ERROR_01 = QRE_IGACR4019E01  # Party with duplicate records
+
+    def __init__(self):
+        AbstractLogicQualityRule.__init__(self)
+
+        self._id = QR_IGACR4019
+        self._name = "Revisar que la tabla interesado no tenga registros repetidos"

```

Comment:
I propose to change the name of the quality rule to: "Interesado no debe tener registros repetidos"

Reply:
Fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/550/commits/f572b3434bf30eb2591e81a157b8b52bd9850fc8